src/styles/_typography.scss

```diff
@@ -1,53 +1,41 @@
-// Styles that go into this file should apply generally to the entire site. Specific component typography should be contained in components.
-
-// Headings
-h1,
-h2,
-h3,
-h4 {
-  // Spacing
-  --flow-spacing: var(--gcds-spacing-600);
-}
+// Styles that go into this file should apply generally to the entire site.
+// Specific component typography should be contained in components.
 
 // Links
 a[href] {
-  color: var(--gcds-link-default);
   text-underline-offset: -0.1em;
   text-decoration-color: currentColor;
 
   &:hover {
     color: var(--gcds-hover-default);
     text-underline-offset: 0em;
-    text-decoration-thickness: 2px;
+    text-decoration-thickness: 0.125rem;
   }
 
   &:focus {
     background-color: var(--gcds-focus-background);
     color: var(--gcds-focus-text);
     text-decoration: none;
-    // box-shadow: 0 0 0 2px var(--gcds-focus-background);
-    outline: 2px solid var(--gcds-focus-background);
+    outline: 0.125rem solid var(--gcds-focus-background);
   }
 }
 
+a[href]:not([class*="link-"]) {
+  color: var(--gcds-link-default);
+}
+
 // Paragraph and long form content
 p {
   font: var(--gcds-font-text-long);
   max-width: 65ch;
 }
 
-p + ol,
-p + ul {
-  // Lists will naturally flow from a paragraph
-  --flow-spacing: 0;
-}
-
```

Comment:
Same idea here. This keeps list closer to the leading paragraph.

Reply:
Same things here, we should be using the utility framework for this.